scripts/recording_uploader/uploader/app.py

```diff
@@ -0,0 +1,40 @@
+"""Lambda function for generating a presigned URL for uploading a recording to S3."""
+
+from typing import Any
+from uuid import uuid4
+import json
+
+from botocore.client import Config
+import boto3
+
+
+def lambda_handler(*args: Any, **kwargs: Any) -> dict:
+    """Main entry point for the lambda function."""
+    return {
+        "statusCode": 200,
+        "body": json.dumps(get_presigned_url()),
+    }
+
+
+def get_presigned_url() -> dict:
+    """Generate a presigned URL for uploading a recording to S3."""
+    bucket = "openadapt"
+    region_name = "us-east-1"
+    s3 = boto3.client(
+        "s3",
+        config=Config(signature_version="s3v4"),
+        region_name=region_name,
+        endpoint_url=f"https://s3.{region_name}.amazonaws.com",
+    )
+    key = f"recordings/{uuid4()}.zip"
```

Comment:
What do you think about adding the user's unique id to the path, e.g. `recordings/{user_id}/{upload_id}.zip`?